auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.java

```diff
@@ -310,4 +361,42 @@ long getCurrentTimeInMillis() {
         return clock.getCurrentTimeMillis();
     }
 
+    private boolean hasScopeChanged(@NonNull String storedScope, @Nullable String requiredScope) {
+        if (requiredScope == null) {
+            return false;
+        }
+        String[] stored = storedScope.split(" ");
+        Arrays.sort(stored);
+        String[] required = requiredScope.split(" ");
+        Arrays.sort(required);
+        return stored != required;
+    }
+
+    private boolean willExpire(Long expiresAt, long minTtl) {
+        if (minTtl == 0 && (expiresAt == null || expiresAt == 0)) {
```

Comment:
Why in this class `willExpire` uses a `Long` for `expiresAt`, while the regular `CredentialsManager` uses a `long` (primitive)? And why the need for this check in this class and not in the other?